src/vl53l0x_driver.cpp

```diff
@@ -136,13 +191,14 @@ int main(int argc, char **argv) {
         sensor_pub_array[i] = nh.advertise<vl53l0x_driver::vl53l0x>(result, 10);
         Sensor_Calibration(pSensors[i]);
       }
-      while (ros::ok() and check_device_connection(pSensors[0]->fd)) {
-        for (int i = 0; i < NUM_SENSORS; i++) {
-          Start_Ranging(i);
-          sensor_pub_array[i].publish(sensor_msg_array[i]);
-          ros::spinOnce();
-        }
-      }
+      ros::AsyncSpinner spinner(4); // Use 4 threads

```

Comment:
we should be able to pass the number of sensors

Reply:
ok